jest.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
     collectCoverageFrom: ['src/**/*.ts', '!src/**/*.spec.ts', '!**/__tests__/**'],
     globals: {
         'ts-jest': {
-            tsConfig: 'tsconfig.spec.json',
+            tsconfig: 'tsconfig.spec.json',
```

Comment:
Doesn't looks to be relevant to me!

Reply:
It’s required due to the `ts-jest` update.